aws-mfa

```diff
@@ -23,38 +23,43 @@ logger.addHandler(stdout_handler)
 logger.setLevel(logging.DEBUG)
 
 AWS_CREDS_PATH = '%s/.aws/credentials' % (os.path.expanduser('~'),)
-STS_DEFAULT = 900
+STS_DEFAULT = 43200  # in seconds. AWS default
```

Comment:
We could create two default variables or, try not to have defaults. If the value is missing, just don't pass this argument, AWS default will be used instead.